src/main/java/seedu/address/logic/parser/TeamCommandsParser.java

```diff
@@ -21,7 +22,8 @@ public class TeamCommandsParser {
     public static final String MESSAGE_USAGE = "Commands for team management:\n"
             + AddTeamCommand.MESSAGE_USAGE + "\n\n"
             + ListTeamCommand.MESSAGE_USAGE + "\n\n"
-            + DeleteTeamCommand.MESSAGE_USAGE;
+            + DeleteTeamCommand.MESSAGE_USAGE
```

Comment:
Double newline needed here for formatting.
```suggestion
            + DeleteTeamCommand.MESSAGE_USAGE + "\n\n"
```